src/py/flwr/server/superlink/fleet/message_handler/message_handler.py

```diff
@@ -124,5 +127,14 @@ def get_run(
             fab_id=run.fab_id,
             fab_version=run.fab_version,
             override_config=user_config_to_proto(run.override_config),
+            fab_hash=run.fab_hash,
         )
     )
+
+
+def get_fab(
+    request: GetFabRequest, ffs: Ffs  # pylint: disable=W0613
+) -> GetFabResponse:
+    """Get FAB."""
+    fab = Fab(request.hash_str, ffs.get(request.hash_str)[0])
```

Comment:
What happens when this fails?

Reply:
@danieljanes I updated the code to make the `get` function of `Ffs` return an optional. I can move this change to another PR if you prefer